scripts/pull_passwords.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+import sys
+import pwsafe
+
+USERNAMES = [
+  'dev_pubcloud_api_key',
```

Comment:
Sure, I'll get these added.  For simplicity (and because not everything is a username/password), I've been storing username and passwords separately as secret text credentials.  I'll need to create two new credentials for rpcghwebhooks and update any JJB references to use those.  Any objections?